.test-infra/jenkins/job_beam_PostCommit_Java_GradleBuild.groovy

```diff
@@ -20,46 +20,50 @@ import common_job_properties
 
 // This job runs the Java postcommit tests, including the suite of integration
 // tests.
-mavenJob('beam_PostCommit_Java_MavenInstall') {
-  description('Runs postcommit tests on the Java SDK.')
-
+job('beam_PostCommit_Java_GradleBuild') {
```

Comment:
As it exists, this is a Dataflow Java PostCommit, rather than a Gradle-target. Perhaps a name change is in order?

Reply:
This also runs the other runners as well via the precommit dependency.

Also, I think this is the place where we do want all postcommits to live, not just Dataflow specific ones (even though they may be the only ones).